.travis.yml

```diff
@@ -51,6 +51,19 @@ stage_linux: &stage_linux
   language: python
   python: 3.5
 
+stage_linux_latex: &stage_linux_latex
+  <<: *stage_linux
+  before_install:
+    - sudo apt-get install -y texlive-latex-base
+    - sudo apt-get install -y texlive-fonts-recommended
+    - sudo apt-get install -y texlive-fonts-extra
+    - sudo apt-get install -y texlive-latex-extra
+    - sudo apt-get install -y poppler-utils
+    - sudo apt-get install -y texlive-pictures
+  script:
+    - wget https://raw.githubusercontent.com/CQuIC/qcircuit/master/qcircuit.sty
```

Comment:
I assume this is needed because the texlive package in xenial (or maybe trusty, I forget what travis uses) is using a qcircuit too old. If that's the case can we add a comment about that.

Also instead of using git master we should use the latest release. You can get that from CTAN here: http://mirrors.ctan.org/graphics/qcircuit/qcircuit.sty
(which is where it gets pulled from for the package)